samples/wifi/shell/prj.conf

```diff
@@ -93,3 +93,5 @@ CONFIG_SETTINGS_NVS=y
 # environment.
 CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
 CONFIG_NET_SOCKETS_POLL_MAX=10
+
+CONFIG_MBEDTLS_SHA1_C=y
```

Comment:
This needs to done at `modules/hostap/Kconfig` so that it applies to all users of Wi-Fi. And also why is this only a problem with OT and not others? By default without OT Wi-Fi works fine.

Reply:
Or as its only for OT, we should move this to `overaly-openthread.conf`

Reply:
Sure, I moved it to the overlay of openthread.